selfdrive/car/gm/carcontroller.py

```diff
@@ -15,6 +15,11 @@ def __init__(self, car_fingerprint):
       self.STEER_STEP = 2              # how often we update the steer cmd
       self.STEER_DELTA_UP = 7          # ~0.75s time to peak torque (255/50hz/0.75s)
       self.STEER_DELTA_DOWN = 17       # ~0.3s from peak torque to zero
+    elif car_fingerprint == CAR.ACADIA_DENALI:
+      self.STEER_MAX = 255
+      self.STEER_STEP = 3              # how often we update the steer cmd
```

Comment:
@rbiasini, @CryptoKylan says stock LKA is at 50 Hz, but with STEER_STEP = 2 it sometimes faults out (hence STEER_STEP = 3).

@CryptoKylan, have you experimented with smaller DELTA_UP but still "STEER_STEP = 2"? What's the highest "delta up" you have seen in stock LKA logs?